source/Irrlicht/CMakeLists.txt

```diff
@@ -127,6 +127,7 @@ set(IRRLICHT_SRCS_COMMON
 	${IRR_ROOT_PATH}/src/irr/asset/CImageLoaderJPG.cpp
 	${IRR_ROOT_PATH}/src/irr/asset/CImageLoaderPNG.cpp
 	${IRR_ROOT_PATH}/src/irr/asset/CImageLoaderTGA.cpp
+	${IRR_ROOT_PATH}/src/irr/asset/CGLILoader.cpp
```

Comment:
removed DDS?

Reply:
@devshgraphicsprogramming do you want me to delete DDS loader?

Reply:
well yeah, no point in duplicate loaders.